### PR TITLE
Update Munin specific configs

### PR DESCRIPTION
**Pull request by MaxUlysse**
- Update Sarek/Munin specific configs
- Add rnafusion/Munin specific configs
- Fix links to config files in pipeline/specific docs
- Markdown lint

### CODE REVIEW

**Comment by MaxUlysse** [comment]
@nf-core/core Not actually sure why, but I add markdownlint issues with the unordered list on my computer, so that's why I changed the `*` by `-`

**Comment by apeltzer** [comment]
> 
> 
> @nf-core/core Not actually sure why, but I add markdownlint issues with the unordered list on my computer, so that's why I changed the `*` by `-`

I had that too - I think the default by markdownlint is to request the `-` though I like the `*` more. I think you can configure your local markdownlint too to tkae that as new default ...?

**Comment by MaxUlysse** [comment]
> > @nf-core/core Not actually sure why, but I add markdownlint issues with the unordered list on my computer, so that's why I changed the `*` by `-`
> 
> I had that too - I think the default by markdownlint is to request the `-` though I like the `*` more. I think you can configure your local markdownlint too to tkae that as new default ...?

I think I have the `-` in sarek now, so I'm kinda used to that one, but actually I don't really care.
I'll revert the `*` to `-` changes I made, and I'll ask for a decision on that particular Markdown formatting matter somewhere else

**Comment by apeltzer** [comment]
For reference, this is the rule that causes the trouble there: 

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md004---unordered-list-style

**Comment by apeltzer** [comment]
Sorry :-( 

I think this is the rule we'd need to adjust in the `markdownlint.yml`:

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md004---unordered-list-style

File: conf/pipeline/sarek/munin.config [hunk 1]
```diff
@@ -1,22 +1,28 @@
-// Profile config names for nf-core/configs
+ // sarek/munin specific profile config
```
Comment (MaxUlysse) [inline conf/pipeline/sarek/munin.config]:
```suggestion
// sarek/munin specific profile config
```

**Comment by MaxUlysse** [comment]
Actually tests are working fine, so no need to change anything...

**Pull request merged**